addons/website_project_issue/controllers/main.py

```diff
@@ -37,6 +37,12 @@ def my_issues(self, page=1, date_begin=None, date_end=None, sortby=None, filterb
         searchbar_filters = {
             'all': {'label': _('All'), 'domain': []},
         }
+        searchbar_inputs = {
```

Comment:
I would avoid having html outside of templates. It makes edition and customization of templates quite hard.